ltiauthenticator/lti11/auth.py

```diff
@@ -0,0 +1,83 @@
+from tornado import gen
+
+from traitlets import Dict
+
+from jupyterhub.auth import Authenticator
+from jupyterhub.utils import url_path_join
+
+from ltiauthenticator.lti11.handlers import LTIAuthenticateHandler
+from ltiauthenticator.lti11.validator import LTILaunchValidator
+
+
+class LTIAuthenticator(Authenticator):
```

Comment:
Let's call this LTI11Authenticator, we can alias it in `__init__.py` with something like `from .lti11.auth import LTI11Authenticator as LTIAuthenticator` for the aliasing

Reply:
Got it. I added the alias in the init but forgot to update the class name.